apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 import { FocusZone } from 'office-ui-fabric-react/lib/FocusZone';
-import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
+import { SearchBox, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
```

Comment:
@micahgodbolt this is mostly because  after changing the `display` property in `IRawStyleBase` from string to the official documentation possible values Travis build were failing here:
![screen shot 2018-10-15 at 10 58 17 am](https://user-images.githubusercontent.com/29514833/46969018-ab5de100-d069-11e8-88de-d9e3c3494fc3.png)